### PR TITLE
fix(daffio): broken assets paths on server

### PR DESCRIPTION
**Pull request by griest024**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/develop/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: apps/daffio/package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "lint": "cd ../.. && ng lint daffio",
     "lint:fix": "npm run lint -- --fix",
     "dev:ssr": "ng run daffio:serve-ssr",
-    "serve:ssr": "node ../../dist/apps/daffio/server/main.js"
+    "serve:ssr": "DAFFIO_DOCS_PATH=http://localhost:4000/assets/daffio/docs/ node ../../dist/apps/daffio/server/main.js"
```
Comment (griest024) [inline apps/daffio/package.json:21]: @damienwebdev should this be committed?

File: apps/daffio/src/app/docs/services/docs.service.ts [hunk 2]
```diff
@@ -1,24 +1,32 @@
...
 
-  constructor(private http: HttpClient) {}
+  constructor(
+    private http: HttpClient,
+    @Inject(DAFFIO_DOCS_PATH_TOKEN) private docsPath: string,
+  ) {}
 
   get(path: string): Observable<T> {
-    return this.http.get<T>(DOCS_BASE_URL + path + '.json');
+    console.log(this.docsPath);
```
Comment (damienwebdev) [inline apps/daffio/src/app/docs/services/docs.service.ts:24]: Remove log.

File: apps/daffio/src/app/docs/services/docs.service.ts [hunk 3]
```diff
@@ -1,24 +1,32 @@
...
+
+    return this.http.get<T>(this.docsPath + path + '.json');
   }
   getGuideList(): Observable<DaffioGuideList> {
-    return this.http.get<V>(DOCS_BASE_URL + 'guides/guide-list.json');
+    console.log(this.docsPath);
```
Comment (damienwebdev) [inline apps/daffio/src/app/docs/services/docs.service.ts:29]: Remove log.

File: apps/daffio/package.json (hunk 1 above)
Comment (damienwebdev) [reply apps/daffio/package.json:21]: I think that's ok given that these are the defaults.

**Review by damienwebdev — CHANGES_REQUESTED** [review]

File: apps/daffio/src/app/docs/services/docs.service.ts (hunk 2 above)
Comment (griest024) [reply apps/daffio/src/app/docs/services/docs.service.ts]: removed

File: apps/daffio/src/app/docs/services/docs.service.ts (hunk 3 above)
Comment (griest024) [reply apps/daffio/src/app/docs/services/docs.service.ts]: removed

**Review by damienwebdev — APPROVED** [review]

**Pull request merged**